src/stravalib/tests/unit/test_model.py

```diff
@@ -353,3 +355,50 @@ def test_subscription_update_deser(self):
             "2011-02-09 21:22:21",
             subupd.event_time.strftime("%Y-%m-%d %H:%M:%S"),
         )
+
+
+from datetime import datetime
+
+# Creating a timezone-aware datetime object
+dt = datetime(2022, 4, 28, 12, 0, tzinfo=timezone.utc)
+
+# Converting datetime to POSIX time
+posix_time = int(dt.timestamp())
```

Comment:
This is a bit hacky but allows us to create t imestamp that is timezone agnostic for us to run locally.